sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/WriteStatsTracker.scala

```diff
@@ -103,5 +103,5 @@ trait WriteJobStatsTracker extends Serializable {
    * to the expected derived type when implementing this method in a derived class.
    * The framework will make sure to call this with the right arguments.
    */
-  def processStats(stats: Seq[WriteTaskStats]): Unit
+  def processStats(stats: Seq[WriteTaskStats], jobCommitDuration: Long): Unit
```

Comment:
Could you update the function description by adding `@param jobCommitDuration`, please?

Reply:
Done